mscr-ui/src/modules/form/index.tsx

```diff
@@ -566,69 +568,67 @@ export default function FormModal({
                     ? SpinnerType.CrosswalkRegistrationModal
                     : modalType == ModalType.RegisterNewMscr
                       ? SpinnerType.CrosswalkCreationModal
-                      : SpinnerType.CrosswalkRevisionModal
-              }
+                      : SpinnerType.CrosswalkRevisionModal}
             ></SpinnerOverlay>
           )}
         </>
-        <div id={'modalTop'}></div>
-        <ModalTitle>
-          {contentType == Type.Schema
-            ? modalType == ModalType.RegisterNewFull
-              ? t('content-form.title.schema-register')
-              : modalType == ModalType.McsrCopy
-                ? t('content-form.title.schema-mscr-copy')
-                : t('content-form.title.schema-revision')
-            : modalType == ModalType.RegisterNewFull
-              ? t('content-form.title.crosswalk-register')
-              : modalType == ModalType.RegisterNewMscr
-                ? t('content-form.title.crosswalk-create')
-                : modalType == ModalType.McsrCopy
-                  ? t('content-form.title.crosswalk-mscr-copy')
-                  : t('content-form.title.crosswalk-revision')}
-        </ModalTitle>
-
-        {(modalType == ModalType.RegisterNewFull ||
-          modalType == ModalType.RevisionFull) &&
-          renderFileDropArea()}
+          <div id={'modalTop'}></div>
+         
+        <div className="row">
+            <div className="col-8">
+            <ModalTitle>
+              {contentType == Type.Schema
+                ? modalType == ModalType.RegisterNewFull
+                  ? t('content-form.title.schema-register')
+                  : modalType == ModalType.McsrCopy
+                    ? t('content-form.title.schema-mscr-copy')
+                    : t('content-form.title.schema-revision')
+                : modalType == ModalType.RegisterNewFull
+                  ? t('content-form.title.crosswalk-register')
+                  : modalType == ModalType.RegisterNewMscr
+                    ? t('content-form.title.crosswalk-create')
+                    : modalType == ModalType.McsrCopy
+                      ? t('content-form.title.crosswalk-mscr-copy')
+                        : t('content-form.title.crosswalk-revision')}
+                </ModalTitle>
+          </div>
+          <div className="col-4">
+                <Button style={{ float: 'right' }} variant="secondaryNoBorder"
```

Comment:
I think we should work towards having all styling done in styled components, so that it's easier to change. Then, when you want to change how a component looks, you can just go to the .styled file of the component and find all style declarations there, instead of some things being inline.